Please fix tests.foreman.cli.test_hammer.HammerCommandsTestCase

There have been some changes to `hammer` which requires an update to our `tests/foreman/data/hammer_commands.json` file. I have already handled all changes related to `max-content-hosts -> max-hosts` and `unlimited-content-hosts -> unlimited-hosts` but there are still a handful to be fixed.

`self = <tests.foreman.cli.test_hammer.HammerCommandsTestCase testMethod=test_positive_all_options>      @tier1     def test_positive_all_options(self):         """check all provided options for every hammer command                  @Feature: Hammer                  @Assert: All expected options are present             """         self.maxDiff = None         self._traverse_command_tree('hammer')         if self.differences:             self.fail( >               '\n' + _format_commands_diff(self.differences)             ) E           AssertionError:  E           hammer activation-key create E             Added options: E               * max-hosts E               * unlimited-hosts E             Removed options: E               * max-content-hosts E               * unlimited-content-hosts E            E           hammer activation-key update E             Added options: E               * max-hosts E               * unlimited-hosts E             Removed options: E               * max-content-hosts E               * unlimited-content-hosts E            E           hammer compute-resource create E             Added options: E               * display-type E            E           hammer compute-resource update E             Added options: E               * display-type E            E           hammer content-view create E             Added options: E               * product E               * product-id E            E           hammer docker registry create E             Added options: E               * organization-ids E               * organizations E               * location-ids E               * locations E            E           hammer docker registry update E             Added options: E               * organization-ids E               * organizations E               * location-ids E               * locations E            E           hammer sc-param update E             Removed options: E               * new-name E            E           hammer template E             Added subcommands: E               * clone E            E           hammer template clone (new command) E             Added options: E               * name E               * help E               * new-name E               * id  tests/foreman/cli/test_hammer.py:145: AssertionError`

--- a/tests/foreman/data/hammer_commands.json
+++ b/tests/foreman/data/hammer_commands.json
@@ -2427,6 +2427,12 @@
               "value": "DESCRIPTION"
             },
             {
+              "help": "for Libvirt only Possible value(s): 'VNC', 'SPICE'",
+              "name": "display-type",
+              "shortname": null,
+              "value": "DISPLAY_TYPE"
+            },
+            {
               "help": "REPLACE locations with given ids Comma separated list of values.",
               "name": "location-ids",
               "shortname": null,
@@ -3019,6 +3025,12 @@
               "name": "description",
               "shortname": null,
               "value": "DESCRIPTION"
+            },
+            {
+              "help": "for Libvirt only Possible value(s): 'VNC', 'SPICE'",
+              "name": "display-type",
+              "shortname": null,
+              "value": "DISPLAY_TYPE"
             },
             {
               "help": "",
@@ -3852,6 +3864,18 @@
               "name": "organization-label",
               "shortname": null,
               "value": "ORGANIZATION_LABEL"
+            },
+            {
+              "help": "Product name to search by",
+              "name": "product",
+              "shortname": null,
+              "value": "PRODUCT_NAME"
+            },
+            {
+              "help": "product numeric identifier",
+              "name": "product-id",
+              "shortname": null,
+              "value": "PRODUCT_ID"
             },
             {
               "help": "Comma separated list of values.",
@@ -7760,10 +7784,34 @@
                   "value": "DESCRIPTION"
                 },
                 {
+                  "help": "REPLACE locations with given ids Comma separated list of values.",
+                  "name": "location-ids",
+                  "shortname": null,
+                  "value": "LOCATION_IDS"
+                },
+                {
+                  "help": "Comma separated list of values.",
+                  "name": "locations",
+                  "shortname": null,
+                  "value": "LOCATION_NAMES"
+                },
+                {
                   "help": "",
                   "name": "name",
                   "shortname": null,
                   "value": "NAME"
+                },
+                {
+                  "help": "REPLACE organizations with given ids. Comma separated list of values.",
+                  "name": "organization-ids",
+                  "shortname": null,
+                  "value": "ORGANIZATION_IDS"
+                },
+                {
+                  "help": "Comma separated list of values.",
+                  "name": "organizations",
+                  "shortname": null,
+                  "value": "ORGANIZATION_NAMES"
                 },
                 {
                   "help": "",
@@ -7896,6 +7944,18 @@
                   "value": "ID"
                 },
                 {
+                  "help": "REPLACE locations with given ids Comma separated list of values.",
+                  "name": "location-ids",
+                  "shortname": null,
+                  "value": "LOCATION_IDS"
+                },
+                {
+                  "help": "Comma separated list of values.",
+                  "name": "locations",
+                  "shortname": null,
+                  "value": "LOCATION_NAMES"
+                },
+                {
                   "help": "Name to search by",
                   "name": "name",
                   "shortname": null,
@@ -7906,6 +7966,18 @@
                   "name": "new-name",
                   "shortname": null,
                   "value": "NEW_NAME"
+                },
+                {
+                  "help": "REPLACE organizations with given ids. Comma separated list of values.",
+                  "name": "organization-ids",
+                  "shortname": null,
+                  "value": "ORGANIZATION_IDS"
+                },
+                {
+                  "help": "Comma separated list of values.",
+                  "name": "organizations",
+                  "shortname": null,
+                  "value": "ORGANIZATION_NAMES"
                 },
                 {
                   "help": "",
@@ -23046,12 +23118,6 @@
               "value": "NAME"
             },
             {
-              "help": "",
-              "name": "new-name",
-              "shortname": null,
-              "value": "NEW_NAME"
-            },
-            {
               "help": "Override this parameter. One of true/false, yes/no, 1/0.",
               "name": "override",
               "shortname": null,
@@ -24793,6 +24859,37 @@
           "description": "Update the default PXE menu on all configured TFTP servers",
           "name": "build-pxe-default",
           "options": [
+            {
+              "help": "print help",
+              "name": "help",
+              "shortname": "h",
+              "value": null
+            }
+          ],
+          "subcommands": []
+        },
+        {
+          "description": "Clone a provision template",
+          "name": "clone",
+          "options": [
+            {
+              "help": "",
+              "name": "id",
+              "shortname": "",
+              "value": "ID"
+            },
+            {
+              "help": "Name to search by",
+              "name": "name",
+              "shortname": "",
+              "value": "NAME"
+            },
+            {
+              "help": "template name",
+              "name": "new-name",
+              "shortname": "",
+              "value": "NEW_NAME"
+            },
             {
               "help": "print help",
               "name": "help",

